core/data_migration_scripts/EE_DMS_Core_4_1_0.dms.php

```diff
@@ -947,7 +947,7 @@ public function insert_default_tickets()
     public function get_or_create_country($country_name)
     {
         if (! $country_name) {
-            throw new EE_Error(esc_html__("Could not get a country because country name is blank", "event_espresso"));
+            throw new EE_Error(esc_html__("Could NOT get a country because country name is blank", "event_espresso"));
```

Comment:
I'm guessing this might have been capitalized as a result of search/replace.  Should it be left lowercase thought so it doesn't break existing string translations?

Reply:
totally, thanks